stat_fs_test.go

```diff
@@ -0,0 +1,33 @@
+package hdfs
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatFs(t *testing.T) {
+	t.Skip("Stat FS does not work on Travis CI")
```

Comment:
We can make this work by

a) making all the other tests clean up after themselves (most should already)
b) assuming a fixed size, rather than shelling out